static/app/views/replays/detail/network/index.tsx

```diff
@@ -0,0 +1,173 @@
+import {Fragment, useCallback, useMemo, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {PanelTable, PanelTableHeader} from 'sentry/components/panels';
+import Placeholder from 'sentry/components/placeholder';
+import {showPlayerTime} from 'sentry/components/replays/utils';
+import {IconArrow} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {EventTransaction} from 'sentry/types';
+import theme from 'sentry/utils/theme';
+
+import {ISortConfig, NetworkSpan, sortNetwork} from './utils';
+
+type Props = {
+  event: EventTransaction;
+  networkSpans: NetworkSpan[];
+};
+
+function NetworkList({event, networkSpans}: Props) {
+  const {startTimestamp} = event;
+  const [sortConfig, setSortConfig] = useState<ISortConfig>({
+    by: 'startTimestamp',
+    asc: true,
+  });
+
+  const handleSort = useCallback(
+    (heading: typeof sortConfig.by) => {
+      setSortConfig(prevSort => {
+        if (prevSort.by === heading) {
+          return {by: heading, asc: !prevSort.asc};
+        }
+
+        return {by: heading, asc: true};
+      });
+    },
+    [sortConfig]
+  );
+
+  const networkData = useMemo(
+    () => sortNetwork(networkSpans, sortConfig),
+    [networkSpans, sortConfig]
+  );
+
+  const columns = [
+    t('status'),
+    t('path'),
+    t('type'),
+    t('duration'),
```

Comment:
Can we add sorting for all the columns? The sort method looks setup to handle most of them already

Reply:
Loom video: 

https://user-images.githubusercontent.com/39612839/180801151-a45b0c76-cf3b-4b45-8c52-e47652d89c3a.mp4